code/game/atom/alternate_appearance.dm

```diff
@@ -99,6 +132,24 @@ GLOBAL_LIST_EMPTY(active_alternate_appearances)
 	if(ghost_appearance)
 		QDEL_NULL(ghost_appearance)
 
+/datum/atom_hud/alternate_appearance/basic/track_mob(mob/new_viewer)
+	. = ..()
+	RegisterSignals(new_viewer, list(
+		COMSIG_MOB_ANTAGONIST_REMOVED,
+		COMSIG_MOB_GHOSTIZED,
+		COMSIG_MOB_MIND_TRANSFERRED_INTO,
+		COMSIG_MOB_MIND_TRANSFERRED_OUT_OF,
+	), PROC_REF(check_hud), override = TRUE)
```

Comment:
I get why you used the other override, but why this one? Shouldn't each alternate_appearance instance not be tracking duplicate mobs?

Reply:
Showing a hud to the same mod twice is valid